bindings/julia/Displaz.jl

```diff
@@ -50,15 +49,54 @@ function write_ply_points(fileName, nvertices, fields)
             for i = 1:size(value,2)
                 propName = ply_property_name(semantic, i)
                 write(fid, "property $typeName $propName\n")
-            end
+        	end
         end
         write(fid, "end_header\n")
         for (_,value) in convertedFields
-            write(fid, value')
+			write(fid, value')
         end
     end
 end
 
+function write_ply_lines(fileName, position, color)
+	vertexValid = ~all(isnan(position),2)
```

Comment:
maybe we live with this for now, and try to figure out a better way of passing multiple line segments as a separate issue.

Did you test the nan functionality?  When I tried the following it didn't result in two line segments like I expected:
```julia
julia> P = [1.0 1 0;
            2 1 0;
       3 1 0;
       NaN NaN NaN;
       1 2 0;
       2 2 0;
       3 2 0]
7x3 Array{Float64,2}:
   1.0    1.0    0.0
   2.0    1.0    0.0
   3.0    1.0    0.0
 NaN    NaN    NaN  
   1.0    2.0    0.0
   2.0    2.0    0.0
   3.0    2.0    0.0

julia> using Displaz
julia> Displaz.plot(P, markershape='-')
```